internal/ingress/metric/collector/collector_test.go

```diff
@@ -60,7 +61,7 @@ func TestNewUDPLogListener(t *testing.T) {
 	conn.Close()
 
 	time.Sleep(1 * time.Millisecond)
-	if count != 1 {
-		t.Errorf("expected only one message from the UDP listern but %v returned", count)
+	if atomic.LoadUint64(&count) != 1 {
+		t.Errorf("expected only one message from the UDP listern but %v returned", atomic.LoadUint64(&count))
```

Comment:
(nit) if you don’t store the value you may print something which is different from what you compared.

Reply:
(typo) "listener"

Reply:
done